src/styles/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 16px;
+
+  &__product-image {
+    padding: 16px 3px 0;
+    display: flex;
+    top: 33px;
+    left: 20px;
```

Comment:
The `top` and `left` properties are not doing anything here because the parent element is not positioned. These properties only work when the element itself or its parent has a position other than `static`.